verso/src/lib.rs

```diff
@@ -3,23 +3,49 @@ use versoview_messages::ControllerMessage;
 
 use ipc_channel::ipc::{IpcOneShotServer, IpcSender};
 
+#[derive(Debug, Default)]
+pub struct VersoviewSettings {
+    pub with_panel: bool,
+}
+
 pub struct VersoviewController(IpcSender<ControllerMessage>);
 
 impl VersoviewController {
-    /// Create a new verso instance and get the controller to it
-    pub fn new(verso_path: impl AsRef<Path>, initial_url: url::Url) -> Self {
+    /// Create a new verso instance with settings and get the controller to it
+    fn create(
+        verso_path: impl AsRef<Path>,
+        initial_url: url::Url,
+        settings: VersoviewSettings,
+    ) -> Self {
         let path = verso_path.as_ref();
         let (server, server_name) =
             IpcOneShotServer::<IpcSender<ControllerMessage>>::new().unwrap();
-        Command::new(path)
+        let mut command = Command::new(path);
+        command
             .arg(format!("--ipc-channel={server_name}"))
-            .arg(format!("--url={initial_url}"))
-            .spawn()
-            .unwrap();
+            .arg(format!("--url={initial_url}"));
+        if !settings.with_panel {
```

Comment:
I prefer to use `no-panel` in the settings and set the panel as default. What do you think?

Reply:
I feel like a webview should not normally have the panel, `with_panel` reflects that and we probably only want that in verso browser